CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## v1.5.1 Release Notes
 
+### All attempts (not just the latest) are now included in the timing diagram tab on the Job Details page.
```

Comment:
optional: I've been thinking how could we make the release notes a bit easier to read, so compared to what we are doing now:

## v1.5.1 Release Notes

### All attempts (not just the latest) are now included in the timing diagram tab on the Job Details page.
### Added a link to the top-level execution directory for the job.
### Added the ability to completely expand the operation details with one click.

It would be more clear to use bullet points:

## v1.5.1 Release Notes

- All attempts (not just the latest) are now included in the timing diagram tab on the Job Details page.
- Added a link to the top-level execution directory for the job.
- Added the ability to completely expand the operation details with one click.

Let alone the later one, instead of making everything bold in `H2`, let you make some words bold/italic and distinguishable, e.g.

- Added a link to the **top-level** execution directory for the job.

Reply:
Just my 2 cents, don't have to change if you don't feel so!

Reply:
A format change to the CHANGELOG should be a separate issue and PR.  Feel free to add a Jira ticket at https://broadworkbench.atlassian.net/secure/RapidBoard.jspa?rapidView=39&projectKey=BA